@xen-orchestra/web/src/components/site/dashboard/BackupIssues.vue

```diff
@@ -8,20 +8,56 @@
     <VtsLoadingHero v-if="!isReady" type="card" />
     <VtsNoDataHero v-else-if="!hasBackupIssues" type="card" />
     <div v-else class="backup-items">
-      <VtsBackupItem v-for="(coreBackupIssue, index) in coreBackupIssues" :key="index" :backup="coreBackupIssue" />
+      <VtsDataTable is-ready>
+        <template #thead>
+          <tr>
+            <template v-for="column of visibleColumns" :key="column.id">
+              <th :class="{ logs: column.id.startsWith('logs-') }">
+                <div v-tooltip class="text-ellipsis">
+                  <VtsIcon accent="brand" :icon="headerIcon[column.id]" />
+                  {{ column.label }}
+                </div>
+              </th>
+            </template>
+          </tr>
+        </template>
+        <template #tbody>
+          <tr v-for="row of rows" :key="row.id">
+            <td v-for="column of row.visibleColumns" :key="column.id">
```

Comment:
column.id is not unique. `column.id: "name" | `logs-${number
```suggestion
            <td v-for="column of row.visibleColumns" :key="column.id=='name' ? column.value : column.id">
```